lib/sendgrid/marketing_campaigns/contacts/recipients.ex

```diff
@@ -24,6 +24,39 @@ defmodule SendGrid.Contacts.Recipients do
     |> handle_recipient_result
   end
 
+  @doc """
+  Adds or updates multiple recipients in contacts list.
+  Recipients param must be in format required by Sendgrid:
+    [
+      %{
+        "email" => "test@example.com",
+        "name"  => "John Doe",
+        etc...
+      }
+    ]
+  """
+  @spec add_multiple([]) :: { :ok, [String.t] } | { :ok, String.t } | { :error, list(String.t) }
```

Comment:
Refer to the comment about the return for this function.

```suggestion
  @spec add_multiple([String.t()]) :: {:ok, map()} | {:error, any()}
```

Reply:
done